pkg/maps/policymap/policymap.go

```diff
@@ -39,6 +40,10 @@ const (
 	// programs. This number *MUST* be identical to the maximum number of
 	// allowed identities.
 	ProgArrayMaxEntries = identity.MaxIdentity
+
+	// AllPorts represents port 0, which logically means that the port is not considered
+	// in PolicyMap lookups; all ports are allowed.
```

Comment:
I found the comment here more confusing than the name. Do you think that something like the below would clarify it?

`All ports is used to ignore the L4 ports in PolicyMap lookups; all ports are allowed. In the datapath, this is represented with the value 0 in the port field of map elements.` ?

Reply:
That is much more well-phrased than what I said, and I am all for more readable documentation. Will post upload shortly with this change.

Reply:
Fixed :)